tests/test_sampler.py

```diff
@@ -419,6 +419,88 @@ def test_sampling_request_callback():
 
     sampler.close()
 
+# noinspection PyProtectedMember
+def test_sampling_request_callback_bytes():
```

Comment:
this seems like an awfully long test just to check the return type. Maybe one of the existing tests can be extended instead?